src/firestore/firekit.ts

```diff
@@ -1178,6 +1180,7 @@ export class RoarFirekit {
     // First add the administration to the database
     const administrationData: Administration = {
       name,
+      publicName: publicName ?? "",
```

Comment:
Maybe if `publicName` isn't provided we should fallback to the `name` param.
```suggestion
      publicName: publicName ?? name,
```